Cargo.toml

```diff
@@ -76,14 +77,25 @@ serde = { version = "1", features = ["derive"] } # used by the plugin example
 typetag = "0.1" # used by the plugin example
 
 [features]
-default = ["fast-tests", "wasi", "backend-cranelift"]
-"loader-kernel" = ["wasmer-kernel-loader"]
+default = ["fast-tests", "wasi", "experimental-io-devices"]
```

Comment:
Removing the default backend again?  This is a change we've been oscillating between and I think it's better for development to have a default backend